extensions/wikia/InsightsV2/helpers/InsightsPageViews.php

```diff
@@ -54,7 +54,9 @@ public function assignPageViewsData( $articlesData ) {
 			$sortingData['pvDiff'][ $articleId ] = $pvDiff;
 			$articlesData[ $articleId ]['metadata']['pvDiff'] = $pvDiff;
 
-			$sortingData['random'][ $articleId ] = rand();
+			$randValue = rand();
```

Comment:
Maybe we should use better random values? http://php.net/manual/en/function.mt-rand.php